### ISSUE TITLE
conan remove can't remove a package revision from a remote

### ISSUE THREAD

**Issue by smoofra** [conan-io/conan#11432]
conan remove seems to be succeeding when I ask it to remove a single package-revision, but afterwards the revision is still there.

### Environment Details 
  * Operating System+version:  Mac OS 12.3.1
  * Conan version: 1.49.0
  * Python version: 3.9.12
  * Artifactory version: artifactory-cpp-ce:7.35.2

### Steps to reproduce 

* Build a project twice and upload to artifactory both times, so there are multiple package revisions on the server.

* Try to delete one of them:

      ➜ ~ conan search -r release -rev 'gmp/6.2.1@#be7f70255d8a31824350b0a8470c4fc2:16354957389c19c3afaa3401962f13f3787d7369'               
      Revisions for 'None' at remote 'release':
      44298138e42d338368d8cdf3445a5c87 (2022-06-08 17:46:39 UTC)
      d985ba89791e96a8b2bdce730f39ad4c (2022-06-08 16:53:44 UTC)
      
      
      ➜ ~ conan remove -r release 'gmp/6.2.1@#be7f70255d8a31824350b0a8470c4fc2:16354957389c19c3afaa3401962f13f3787d7369#d985ba89791e96a8b2bdce730f39ad4c'  

* It's still there:

      ➜ ~ conan search -r release -rev 'gmp/6.2.1@#be7f70255d8a31824350b0a8470c4fc2:16354957389c19c3afaa3401962f13f3787d7369'               
      Revisions for 'None' at remote 'release':
      44298138e42d338368d8cdf3445a5c87 (2022-06-08 17:46:39 UTC)
      d985ba89791e96a8b2bdce730f39ad4c (2022-06-08 16:53:44 UTC)

**Comment by memsharded** [comment]
Hi @smoofra 

This seems a small UI issue, and in Conan 1.X it is still necessary to do:
```bash
conan remove -r=release gmp/6.2.1@#be7f70255d8a31824350b0a8470c4fc2 -p 16354957389c19c3afaa3401962f13f3787d7369#d985ba89791e96a8b2bdce730f39ad4c
```

It also seems that this issue has already been solved in Conan 2.0 (and released in previous alphas)

**Comment by smoofra** [comment]
oh, thanks!

**Comment by memsharded** [comment]
This seems to be solved in 2.0, and a workaround with -p syntax is possible in 1.X, closing. Thanks for the feedback!
